tests/ssl-opt.sh

```diff
@@ -2275,6 +2282,7 @@ run_test    "Opaque key for server authentication: invalid alg: ECDHE-ECDSA with
 
 requires_config_enabled MBEDTLS_USE_PSA_CRYPTO
 requires_config_enabled MBEDTLS_X509_CRT_PARSE_C
+requires_config_enabled MBEDTLS_RSA_C
```

Comment:
This looks redundant with the auto-detecting based on server2 / server7 added above. What am I missing?

Reply:
This looks like a left over. It's probably the 1st error which I tried to solve, then I added the general rule while forgetting to remove this line. Since it seems you don't have any major concerns about this PR, if you don't mind, I would like to fix this error once I will do the final rebase after the preceding PRs will be merged.

Reply:
>  I would like to fix this error once I will do the final rebase after the preceding PRs will be merged.

Sure!

Reply:
Done here https://github.com/Mbed-TLS/mbedtls/pull/8008/commits/522ce71be56dc7ed041014c380c6af7010aed577

Reply:
Ah, should have mentioned: two other occurrences (lines 2305 and 2325).

Reply:
Correct. I updated the last commit